cmd/ctr/commands/namespaces/namespaces.go

```diff
@@ -174,10 +174,13 @@ var removeCommand = &cli.Command{
 					log.G(ctx).WithError(err).Errorf("unable to delete %v", target)
 					continue
 				}
-
+				if errdefs.IsNotFound(err) {
+					log.G(ctx).WithError(err).Warnf("namespace %v not found", target)
+					continue
+				}
 			}
 
-			fmt.Println(target)
+			log.G(ctx).Infof("namespace %v deleted", target)
```

Comment:
This will also change the output when a namespace is deleted. Instead of just getting the namespace name on stdout, we will have log message.